.gitignore

```diff
@@ -124,4 +124,5 @@ fabric.properties
 # End of https://www.gitignore.io/api/eclipse,intellij
 /.idea/
 /target/
-/Spaced.Repetition.iml
\ No newline at end of file
+/Spaced.Repetition.iml
+/orlp.log
```

Comment:
Please also add src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt

Reply:
added to .gitignore